mesonbuild/mcompile.py

```diff
@@ -74,8 +76,10 @@ def get_parsed_args_vs(options: 'argparse.Namespace', builddir: Path):
     
     if options.load_average:
         mlog.warning('Msbuild does not have a load-average switch, ignoring.')
+    if options.verbose:
+        cmd.append('/v:detailed')
```

Comment:
I think we agreed that by default vs backend is already verbose enough, and thus do:
```
if not options.verbose:
  cmd.append('/v:minimal')
```

Reply:
Done.